aspnetcore/razor-components/index.md

```diff
@@ -1,65 +1,34 @@
-
-![Razor Components runs .NET code on the server and interacts with the Document Object Model on the client over a SignalR connection](index/_static/aspnet-core-razor-components.png)
-
-For more information, see <xref:razor-components/hosting-models#server-side-hosting-model>.
-
-### Client-side hosting model
-
-Running .NET code inside web browsers is made possible by a relatively new technology, [WebAssembly](http://webassembly.org) (abbreviated *wasm*). WebAssembly is an open web standard and is supported in web browsers without plugins. WebAssembly is a compact bytecode format optimized for fast download and maximum execution speed.
-
-WebAssembly code can access the full functionality of the browser via JavaScript interop. At the same time, WebAssembly code runs in the same trusted sandbox as JavaScript to prevent malicious actions on the client machine.
-
-![Blazor runs .NET code in the browser with WebAssembly.](index/_static/blazor.png)
-
-When a Blazor app is built and run in a browser:
-
-1. C# code files and Razor files are compiled into .NET assemblies.
-1. The assemblies and the .NET runtime are downloaded to the browser.
-1. Blazor uses JavaScript to bootstrap the .NET runtime and configures the runtime to load required assembly references. Document object model (DOM) manipulation and browser API calls are handled by the Blazor runtime via JavaScript interoperability.
-
-To support older browsers that don't support WebAssembly, you can use the ASP.NET Core Razor Components [server-side hosting model](#server-side-hosting-model).
-
-For more information, see <xref:razor-components/hosting-models#client-side-hosting-model>.
+* Build rich interactive UIs using .NET instead of JavaScript.
+* Write client-side code and share server and client-side logic among apps.
+* Razor Components render the UI as HTML markup and JavaScript for wide browser support, including mobile browsers.
 
 ## Components
 
-Apps are built with *components*. A component is a piece of UI, such as a page, dialog, or data entry form. Components can be nested, reused, and shared between projects.
+Apps are built with *components*. A component is a piece of UI, such as a page, dialog, or data entry form. Components handle user events and define flexible UI rendering logic. Components can be nested and reused.
 
-A *component* is a .NET class. The class can either be written directly, as a C# class (*\*.cs*), or more commonly in the form of a Razor markup page (*\*.cshtml*).
+Components are .NET classes built into .NET assemblies that can be shared and distributed as NuGet packages. The class can either be written in the form of a Razor markup page (*.cshtml*) or as a C# class (*.cs*).
```

Comment:
This is fine for now, but in the next preview we'll switch to .razor files instead of .cshtml files for Razor Components.